### ISSUE TITLE
CSS zoom causes XYCursor to miscalculate its position

### ISSUE THREAD

**Issue by martynasma** [amcharts/amcharts4#686]
Looks like determining position does not count in the zoom. Causing cursor to be slightly off to left and right-most sections of the chart not accessible.

Example:
https://codepen.io/team/amcharts/pen/4bee0af66fac24f7ebf547bc2ef8cacc/

**Comment by zeroin** [comment]
It is not recommended to use css zoom as it is quite unpredictable: https://developer.mozilla.org/en-US/docs/Web/CSS/zoom 

Instead, you should use css transform scale like:

```
#chartdiv {    
  transform: scale(0.8);
}```

Although current release will result quite the same issue like with css zoom, but it will be fixed in next release (css zoom issue won't be fixed at all).

**Comment by martynasma** [comment]
Fixed in [4.0.15](https://github.com/amcharts/amcharts4/releases/tag/4.0.15).

## [4.0.15] - 2019-01-04

### Added
- `color` property added to `LegendDataItem`. Can be used for coloring `label` or `valueLabel` text like `text = "[{color}] {name}"`.
- `minZoomCount` added to `Component`.
- Export now allows formatting numberic values as durations. Use newly added `durationFields` and `durationFormat` settings.

### Changed
- Export: When exporting date to XLSX, date fields will now export as true date fields in Excel (which in turn will format them according to regional settings), unless `useLocal` is set to `false`.

### Fixed
- Charts were not working properly in FireFox Extended Support Release.
- Some computer setups were not properly registering mouse movement.
- `legendSettings` was being ignored initially if legend was in external container.
- "Week year" (format code `"YYYY"`) was not being properly calculated.

[Full change log](https://github.com/amcharts/amcharts4/blob/master/dist/es2015/CHANGELOG.md).

Make sure you clear your browser cache after upgrading. And feel free to contact us again if you are still experiencing this issue.
